api/toko-rest-api.yaml

```
@@ -292,16 +292,14 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/ErrorDetails'
-
-  /users/{userId}:
     delete:
       tags:
         - Users
       summary: Delete user by ID
       description: For valid response try integer IDs with positive integer value. Negative or non-integer values will generate API errors
       operationId: deleteUser
       parameters:
-        - name: userId
+        - name: id
           in: path
           description: ID of the user that needs to be deleted
```

You can refer to the defined `EntityId` parameter:
```suggestion
        - $ref: '#/components/parameters/EntityId'
```